package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "next-update",
   "description": "Tests if module's dependencies can be updated to the newer version without breaking the tests",
-  "version": "0.0.0",
+  "version": "3.5.3",
```

Comment:
please remove this - this module is using `semantic-release` to set the next version automatically